### PR TITLE
Clear db

### PR DESCRIPTION
**Pull request by Krishnakumarskr**
Clear the DB, before exporting the contract to DB.
It will clear the contract_addresses and vaults table!

### CODE REVIEW

File: packages/contracts/script/utils/exporter.js [hunk 1]
```diff
@@ -75,6 +76,24 @@ async function exportToSupabase(dataToStoreOnDB) {
   }
 }
 
+async function clearDB() {
```
[the remark this answers is not in the document]
Comment (jplodge-pro) [reply packages/contracts/script/utils/exporter.js:79]: UPDATE: I also think consistent style is important. For example, the other function is called `exportAddress` and not `exportAddr`.

File: packages/contracts/script/utils/exporter.js [hunk 2]
```diff
@@ -75,6 +76,24 @@ async function exportToSupabase(dataToStoreOnDB) {
...
+  console.log('Crearing DB......')
+  const client = createClient(process.env.NEXT_PUBLIC_SUPABASE_URL, process.env.SUPABASE_SERVICE_ROLE_KEY);
+
+  const contractAddresses = await client.from('contracts_addresses').delete().neq('id', 0);
```
[the remark this answers is not in the document]
Comment (Krishnakumarskr) [reply packages/contracts/script/utils/exporter.js:83]: The id's will not start with 0. This is a condition to delete all rows in the table

[the remark this answers is not in the document]
Comment (jplodge-pro) [reply packages/contracts/script/utils/exporter.js:83]: So a condition is required for a valid Supabase query? 
```client.from('contracts_addresses').delete()``` would not work?

File: packages/contracts/script/utils/exporter.js [hunk 3]
```diff
@@ -75,6 +76,24 @@ async function exportToSupabase(dataToStoreOnDB) {
...
+  if (contractAddresses.error) {
+    console.log(contractAddresses.error);
```
[the remark this answers is not in the document]
Comment (jplodge-pro) [reply packages/contracts/script/utils/exporter.js:85]: My example isn't great, to be fair! "Failed to truncate 'contract_addresses'." would be better :)

File: packages/contracts/script/utils/exporter.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Krishnakumarskr) [reply packages/contracts/script/utils/exporter.js:84]: Yes, there should be a where clause to the delete query

**Review by jplodge-pro — APPROVED** [review]

**Pull request merged**